modules/rosetta/src/main/scala/org/tessellation/rosetta/domain/construction/ConstructionService.scala

```diff
@@ -48,5 +58,54 @@ object ConstructionService {
 
       NonEmptyList.fromList(accountIdentifiers)
     }
+
+    def transactionToOperations(transaction: Transaction): EitherT[F, ConstructionError, NonEmptyList[Operation]] = {
+      val result = for {
+        transactionAmount <- refineV[Not[Equal[0L]]](transaction.amount.value.value)
+        negativeTransactionAmount <- refineV[Not[Equal[0L]]](-1L * transaction.amount.value.value)
+        negativeOperation = Operation(
+          OperationIdentifier(OperationIndex(0L)),
+          none,
+          OperationType.Transfer,
+          none,
+          AccountIdentifier(transaction.destination, none).some,
```

Comment:
Shouldn't it be `transaction.source`? I mean if you transfer `100 DAG` from `A` to `B` then it probably should be:
`Operation(index = 0, address = 'A', value = -100), Operation(index = 1, address = 'B', value = +100)`.